src/test/unit/requests/channelManagementRequests.test.ts

```diff
@@ -32,22 +32,22 @@ describe('Channel Management requests', () => {
 
         it('should return media applications', async () => {
             getStub.withArgs(getStoredFilesEndpoint).resolves({ data: dummyData });
-            expect(await channelManagementRequests.getStoredFiles('abc')).toEqual(dummyData);
+            expect(await channelManagementRequests.getStoredFiles('abc', { 'x-user-id': '1234' })).toEqual(dummyData);
         });
 
         it('should return empty array and an error response if get fails', async () => {
             getStub.withArgs(getStoredFilesEndpoint).rejects(errorResponse);
-            expect(await channelManagementRequests.getStoredFiles('abc')).toEqual(null);
+            expect(await channelManagementRequests.getStoredFiles('abc', { 'x-user-id': '1234' })).toEqual(null);
         });
 
         it('should return empty array and an error response if request fails', async () => {
             getStub.withArgs(getStoredFilesEndpoint).rejects(errorRequest);
-            expect(await channelManagementRequests.getStoredFiles('abc')).toEqual(null);
+            expect(await channelManagementRequests.getStoredFiles('abc', { 'x-user-id': '1234' })).toEqual(null);
         });
 
         it('should return empty array and an error response if request fails', async () => {
```

Comment:
I feel like this text (and the one above should change the description name to 'null and an error response' rather than an empty array

Reply:
Updated